.github/workflows/github-page.yml

```diff
@@ -124,10 +124,6 @@ jobs:
     - name: Build Haddock documentation 🔧
       run: ./.github/bin/haddocks.sh ./haddocks true
 
-    - name: View HTML files
-      run: |
-        find "dist-newstyle/build" -name '*.html' | xargs grep html
```

Comment:
This had no effect besides outputting a big number of lines to the pipeline's log.